config/301-kafkatarget.yaml

```diff
@@ -233,6 +236,7 @@ spec:
                         type: boolean
                 required:
                 - saslEnable
+                - tlsEnable
```

Comment:
Should be okay to make this optional. Just so long as omitempty is set on the type.